contrib/lang/clojure/config.el

```diff
@@ -0,0 +1,17 @@
+;;; config.el --- Configuration variables specific to
+;;;               the Clojure configuration layer
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(defvar spacemacs-clojure-fancify-symbols nil
```

Comment:
rename it to `clojure-enable-fancify-symbols` to be consistant with the other layer variables (like in git layer, the erlang-elixir one has to be updated)